tests/sources/fixtures/salesforce/fixture.py

```diff
@@ -170,7 +163,7 @@ def token():
 def query(version):
     query = request.args.get("q")
     table_name = re.findall(r"\bFROM\s+(\w+)", query)[-1]
-    done = random.choice([True, False])
+    done = False
 
     response = {"done": done, "records": generate_records(table_name)}
```

Comment:
Does this handle pagination or do we just ignore pagination for ftests? Asking because the final response should be `done: True`, but I can see we handle pagination now by the presence of `nextRecordsUrl` in the response. 

Also nit: there's no point declaring `done` anymore if it's always `False`. Might as well just have
```
response = {"done": False, "records": generate_records(table_name)}
```

Reply:
This handles pagination for each objects and the final response will surely have `done: True` to terminate the pagination. Updated the code as well 👍